workers/run_tests.sh

```diff
@@ -11,11 +11,19 @@ cd $script_directory
 # move up a level
 cd ..
 
+# Set up the test data volume directory if it does not already exist
+volume_directory="$script_directory/test_volume"
+if [ ! -d "$volume_directory" ]; then
+    mkdir $volume_directory
```

Comment:
Seems like an extreme edge case, I can handle it if you think it's important but I would think that `mkdir` would give an error message and the user could then remove or rename their `test_volume` file.